build.sh

```diff
@@ -55,15 +54,15 @@ do
         build_configuration=Release
         shift 1
         ;;
-        --restore)
+        --restore|-r)
```

Comment:
Did you consider allowing `-restore` as well? That would mean there was at least one argument form that worked across windows and linux (build.cmd and build.sh respectively). Alternatively we could try and make the `--` syntax work in Windows as well.

Reply:
Yeah 🙁. This is a very ugh-inducing catch-22. The very, very standard Linux (unix?) convention is that single-dash arguments can be combined (something that I *should* implement here) - for example, `./build.sh -rbt` == `./build.sh -r -b -t`). So theoretically, `./build.sh -restore` is equivalent to `./build.sh -r -e -s -t -o -r -e`, which is... not what one wants. However, the Windows convention here is single-dash, which as you say, hopefully should be identical across platforms... 😢 

The best thing would probably be to just say screw linux convention and make `-arg`==`--arg`, which makes me a sad panda, but oh well.

Reply:
> The best thing would probably be to just say screw linux convention and make -arg==--arg, which makes me a sad panda, but oh well.

Hmm. Lets keep it as is for now. We can revisit later when we discuss other unification aspects.